src/containers/Header/Search.tsx

```diff
@@ -170,16 +176,60 @@ export const Search = ({ callback = () => {} }: SearchProps) => {
   const onKeyDown: KeyboardEventHandler<HTMLInputElement> = (event) => {
     if (event.key === 'Enter') {
       handleSearch(event.currentTarget?.value?.trim())
+      setCurrentSearchInput('')
     }
   }
 
+  const [isVisible, setIsVisible] = useState(true)
+
+  useEffect(() => {
+    const timeoutId = setTimeout(() => {
+      setIsVisible(false)
+    }, 10000) // Disappear after 10 seconds
+
+    return () => clearTimeout(timeoutId)
+  }, [])
+
   return (
-    <div className="search">
-      <input
-        type="text"
-        placeholder={t('header.search.placeholder')}
-        onKeyDown={onKeyDown}
-      />
+    <div>
+      <ReactTooltip
+        className="banner"
+        id="my-tooltip-styles"
+        place="top"
+        isOpen={isVisible}
+        clickable
+      >
+        <div className="banner-content">
+          <div>
+            Token search by name and account is now available! Try searching for
+            USD
+          </div>
+          <button
+            className="banner-button"
+            type="button"
+            onClick={() => setIsVisible(false)}
+          >
+            <img src={CloseIcon} width={10} height={10} />
+          </button>
+        </div>
+      </ReactTooltip>
+      <div data-tooltip-id="my-tooltip-styles" className="search">
+        <input
+          type="text"
+          placeholder={t('header.search.placeholder')}
+          onKeyDown={onKeyDown}
+          value={currentSearchInput}
+          onChange={(e) => setCurrentSearchInput(e.target.value)}
+        />
+        {process.env.VITE_ENVIRONMENT === 'mainnet' && (
+          <div className="search-results">
```

Comment:
This is only rendered for mainnet but appears to always be rendered.  Does this mean every page is connecting to xrpl-meta even though it may not need to?

Reply:
resolved now with the caching and API calls over a web socket.